controllers/ccruntime_controller.go

```diff
@@ -675,6 +681,10 @@ func (r *CcRuntimeReconciler) processDaemonset(operation DaemonOperation) *appsv
 			Name:  "SNAPSHOTTER_HANDLER_MAPPING",
 			Value: strings.Join(snapshotter_handler_mapping, ","),
 		},
+		{
+			Name:  "PULL_TYPE_MAPPING",
+			Value: strings.Join(pull_type_mapping, ","),
+		},
```

Comment:
This is outside my comfort zone, but it looks consistent with how snapshotter is treated.